src/Administration/Resources/views/administration/index.html.twig

```diff
@@ -5,7 +5,7 @@
         window._features_ = {{ features|json_encode|raw }};
 
         window.__sw__ = {
-            assetPath: '{{ asset('', 'asset') }}',
```

Comment:
the package name `asset` does not exist, so it could also be left out, to save some computing

Reply:
I'm not to sure about that. Have you checked that with a local CDN? Minio, S3?